src/ParsedText.js

```diff
@@ -63,6 +65,7 @@ class ParsedText extends React.Component {
         <ReactNative.Text
           key={`parsedText-${index}`}
           {...props}
+          {...this.props.childrenProps}
```

Comment:
This should go before the other spread to avoid any potential conflicts, where the child prop might override a required value from the text extraction properties.